src/transformers/audio_utils.py

```diff
@@ -155,7 +160,10 @@ def mel_filter_bank(
         norm (`str`, *optional*):
             If `"slaney"`, divide the triangular mel weights by the width of the mel band (area normalization).
         mel_scale (`str`, *optional*, defaults to `"htk"`):
-            The mel frequency scale to use, `"htk"` or `"slaney"`.
+            The mel frequency scale to use, `"htk"`, `"kaldi"` or `"slaney"`.
+        use_torchaudio_version (`bool`, *optional*, defaults to `False`):
```

Comment:
Not a huge fan of this variable name - would it be better to use a name that describes what operation it induces? And also a docstring that accompanies. From looking at the args and docstring alone, I'm not sure a user would know what this argument enables